terraform/hcl2json_test.go

```diff
@@ -531,48 +532,115 @@ block "label_one" {
 }
 
 func TestExtractVariablesSuccess(t *testing.T) {
+	type TestInput struct {
+		fileName    string
```

Comment:
nit: Do we need the `fileName` here? It was defaulting to `"test"` before and the simpler this test table is the better

Reply:
We added it to prepare for the next ticket where we will have different logic for `.tf` and `.tfvars` files.